benchmark/_benchmark_progress.js

```diff
@@ -40,7 +40,7 @@ class BenchmarkProgress {
     this.completedConfig = 0;
     // Total number of configurations for the current file
     this.scheduledConfig = 0;
-    this.interval = 0;  // result of setInterval for updating the elapsed time
+    this.interval = 0;  // Result of setInterval for updating the elapsed time
```

Comment:
Capitalizing looks weird to me here because this isn't a sentence...

Reply:
I updated the comment and removed the default value as it was misleading.